lib/mic.js

```diff
@@ -11,6 +11,7 @@ var mic = function mic(options) {
     var bitwidth = options.bitwidth || '16';
     var encoding = options.encoding || 'signed-integer';
     var rate = options.rate || '16000';
+    var bufferSize = options.bufferSize || '16384';
```

Comment:
Because the buffer arg for arecord defaults to the minimum of max possible buffer size & 500ms, we can't reliably say we know what a good default value would be.

One alternative is to remove defaults and only add the buffer argument if the bufferSize arg is passed.